scipy/stats/morestats.py

```diff
@@ -1464,9 +1469,8 @@ def ansari(x, y):
 
     Notes
     -----
-    The p-value given is exact when the sample sizes are both less than
-    55 and there are no ties, otherwise a normal approximation for the
-    p-value is used.
+    Exact test should be used when the sample sizes are both less than 55 and
```

Comment:
"should be" is maybe not the right phrasing, this is what ``'auto'`` does but that's not necessarily much better or worse than putting the cutoff at 50 or 60.